docs/recipes/DisplayNameComponents.md

```diff
@@ -0,0 +1,31 @@
+# displayName on React Components
+
+Sometimes, you need to have a explicit displayName on your components and not the default `FelaComponent` for debugging, or maybe for things like storybook. Right now, there isn't an automatic way to do this.
```

Comment:
an explicit displayName for your components rather than the default `FelaComponent` ...
... there is no automatic